src/pages/Swap/index.tsx

```diff
@@ -143,7 +145,12 @@ export default function SwapPage({ className }: { className?: string }) {
       <PageWrapper>
         <Swap
           className={className}
-          chainId={connectedChainId}
+          chainId={
```

Comment:
this logic should be removed, we shouldn't default to mainnet for unsupported chains, we disable the inputs. see the avax PR https://github.com/Uniswap/interface/pull/6905/files#diff-d2d2d34074b22ac06988a155c74cca892a97f5bb4e343d933a1a4f357d4bf888

Reply:
i think we still want to pass chainId=1 so that the swap component knows which network to switch to